js/preload/preloadMain.js

```diff
@@ -209,7 +209,15 @@ function createAPI() {
          * see: https://electron.atom.io/docs/api/desktop-capturer/
          * for interface: see documentation in desktopCapturer/getSources.js
          */
-        getMediaSources: getMediaSources
+        getMediaSources: getMediaSources,
+
+        showAlertSettings: function() {
```

Comment:
Changing this method here i'll also need to update the client app. Shall i make this change and create a PR in client app?